server/models/campaign.py

```diff
@@ -166,6 +167,15 @@ def as_dict(self):
             self, recurse=False, exclude=[Note.room, Note.location, Note.user]
         )
 
+class Marker(BaseModel):
+    uuid = TextField(primary_key=True)
```

Comment:
I would make the uuid a ForeignKeyField of Shape, as that's what it ultimately is.

Also, right now you'll get a rejection from the database if another user tries to set a marker on a shape someone else has already marked as uuid is a primary key.